### PR TITLE
PR: Fix control-click URL in tutorial ValueError

### LINKED ISSUE #9849: Control-click URL in tutorial causes ValueError
### What steps will reproduce the problem?

In the Spyder tutorial, the 'Recommended first steps for Python beginners' section, control-left click on the 'IPython interpreter' link.

I did this because I thought it might open the URL in a new window.  It opens the URL successfully (in my web browser, same as a normal left-click), but it issues the "Spyder has encountered an internal problem!" bug report page.

I'm using windows 10, 64-bit.  Default browser is Firefox.

### Traceback
```python-traceback
  File "C:\Users\mckay\Anaconda3\lib\site-packages\spyder\widgets\browser.py", line 168, in createWindow
    webbrowser.open(to_text_string(self.url().toString()))
  File "C:\Users\mckay\Anaconda3\lib\webbrowser.py", line 78, in open
    if browser.open(url, new, autoraise):
  File "C:\Users\mckay\Anaconda3\lib\webbrowser.py", line 582, in open
    os.startfile(url)
ValueError: startfile: filepath too long for Windows
```

## Versions

* Spyder version: 3.3.1 
* Python version: 3.7.0
* Qt version: 5.8.0
* PyQt5 version: 5.8.2
* Operating System: Windows 10

### Dependencies

```
pyflakes >=0.6.0  :  2.0.0 (OK)
pycodestyle >=2.3 :  2.4.0 (OK)
pygments >=2.0    :  2.2.0 (OK)
sphinx >=0.6.6    :  1.7.9 (OK)
rope >=0.9.4      :  0.11.0 (OK)
jedi >=0.9.0      :  0.12.1 (OK)
nbconvert >=4.0   :  5.4.0 (OK)
pandas >=0.13.1   :  0.24.2 (OK)
numpy >=1.7       :  1.15.1 (OK)
sympy >=0.7.3     :  1.1.1 (OK)
cython >=0.21     :  0.28.5 (OK)
qtconsole >=4.2.0 :  4.4.1 (OK)
IPython >=4.0     :  6.5.0 (OK)
matplotlib >=2.0.0:  3.0.3 (OK)
pylint >=0.25     :  2.1.1 (OK)
```

### PR DESCRIPTION
**Pull request by steff456**
## Description of Changes

* [ ] Wrote at least one-line docstrings (for any new functions)
* [ ] Added unit test(s) covering the changes (if testable)

* [ ] Included a screenshot or animation (if affecting the UI, see [Licecap](https://www.cockos.com/licecap/))

Add a try catch to prevent the ValueError in tutorial

### Issue(s) Resolved

Fixes #9849

### Affirmation

By submitting this Pull Request or typing my (user)name below,
I affirm the [Developer Certificate of Origin](https://developercertificate.org)
with respect to all commits and content included in this PR,
and understand I am releasing the same under Spyder's MIT (Expat) license.

I certify the above statement is true and correct: Steff456

### CODE REVIEW

**Comment by ccordoba12** [comment]
> We could probably leave the import outside the try/except as the thing causing problems is the call to open

I agree with this.

> We should probably add a regression test for this.

I don't think this is necessary. Making a test for could be very time consuming and there's very little to gain with it.

**Comment by goanpeca** [comment]
>I don't think this is necessary. Making a test for this could be very time consuming and there's very little to gain with it.

The test can be as simple as this:

```python
def test_webview_open_regression(qtbot):
    # See: spyder-ide/spyder#....
    wb = WebView()
    qtbot.addWidget(wb)
    wb.setUrl('problem_url')
    wb.createWindow(webwindowtype)
```

Without the fix it should fail (because of the uncaught exception)

**Comment by ccordoba12** [comment]
@goanpeca, your proposed test checks at the end the functionality of the `webbrowser` module, which I'm sure is tested elsewhere.

So I'd say we should skip testing this and move to more important things. I think we really don't need to invest time to test every minor change we do.

**Comment by goanpeca** [comment]
>So I'd say we should skip testing this and move to more important things. I think we really don't need to invest time to test every minor change we do.

Sure that is fine, @ccordoba12 .

@steff456 please undo the commit that include the testing and add a comment to point to the issue (added a comment).

---

One suggestion though, we should merge these type of PRs right away, since "waiting for tests to pass" does not really check anything.

File: spyder/widgets/browser.py [hunk 1]
```diff
@@ -168,7 +168,10 @@ def zoom_in(self):
     #------ QWebEngineView API -------------------------------------------------------
     def createWindow(self, webwindowtype):
         import webbrowser
-        webbrowser.open(to_text_string(self.url().toString()))
+        try:
```
Comment (goanpeca) [inline spyder/widgets/browser.py]:
```suggestion
        # See: spyder-ide/spyder#9849
        try:
```

**Comment by ccordoba12** [comment]
> One suggestion though, we should merge these type of PRs right away, since "waiting for tests to pass" does not really check anything.

I think if we merge right away PRs will be flagged as failing, and I wouldn't like to give that impression to people following the project.

I'll test it with this one to see if that's the case or not.

**Pull request merged**